addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/BodyGenerator.java

```diff
@@ -176,7 +178,7 @@ private String generateJsonPrimitiveValue(Schema<?> schema) {
             return schema.getAnyOf().get(0);
         }
         // Should not be reached, allOf schema is resolved by the parser
-        LOG.error("Unknown composed schema type: {}", schema);
+        LOG.error("Unknown composed schema type: " + schema);
```

Comment:
Should be reverted.

Reply:
Pattern search that I should have performed earlier revealed only one other incident of this formatting change. This should be the last.